component/doc/README.md

```diff
@@ -0,0 +1,346 @@
+#Configuring LinkedIn Authenticator
```

Comment:
There should be a space between the # and the title so that it will appear formatted like this:

# Configuring LinkedIn Authenticator

Please apply this to add headings and subheadings